app/web/features/search/SearchBox.tsx

```diff
@@ -55,42 +56,47 @@ export default function SearchBox({
   const { t } = useTranslation([GLOBAL, SEARCH]);
   const classes = useStyles();
 
-  const { control, setValue, errors, register, handleSubmit } = useForm({
+  const { control, errors } = useForm({
     mode: "onChange",
   });
 
-  function handleOnChangeAutocomplete(event: "" | GeocodeResult) {
+  const handleOnChangeAutocomplete = (event: GeocodeResult) => {
     if (event) {
       setLocationResult(event);
     }
-  }
+  };
 
-  function handleOnChangeKeyword(event: React.ChangeEvent<HTMLInputElement>) {
+  const handleOnChangeKeyword = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
     setQueryName(event.target.value);
     setLocationResult({
       ...locationResult,
-      location: undefined,
+      location: new LngLat(0, 0),
```

Comment:
is the behavior here the same?

Reply:
You're right, I changed this back to undefined and updated the type to allow it. Sorry about that!

I got a little trigger happy last night...

Rather than making just the location undefined though, I'm returning the locationResult as undefined. That's what you had in the type originally anyway.

I did a lot of testing on the map and all seems to be working okay.